### DISCUSSION TITLE
Issues opening in .github Repo

### DISCUSSION THREAD

**Discussion by donavanbecker** [community/community#75740] [Mobile]
When submitting an issue on this repo: https://github.com/homebridge/homebridge

Since it is a shared issue template it takes me to the .github repo: https://github.com/homebridge/.github/issues/new?template=bug_report.yml

Should take me here: https://github.com/homebridge/homebridge/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml

GitHub App: 1.139.0
iOS: 17.2 beta

**Comment by HenSquared** [comment]
Hi @donavanbecker. Thanks for reporting this! It looks like we need to improve our support of Issue forms on Mobile.  I have opened an internal issue for us to discuss this.

Unfortunately, I don't have a better workaround for now other than to leverage the mobile web browser. Sorry about that!

To do this:
Navigate to the repository
Tap the kebab [...] menu in top right corner
Tap 'share via...'
Tap 'open in safari' 
THis opens the repository in the web browser
Tap issues tab
Tap New issue
If you are not logged in, it will ask you to log in
After logging in, select your issue template / issue form.

**Reply by donavanbecker** [reply]
@HenSquared Thanks! No big deal, just figured I would report, Thanks!
